bookmark.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Quiz App</title>
+  </head>
+  <body>
+    <header>
+      <h1>HARRY POTTER<br /><em>Quiz App</em></h1>
+    </header>
+    <main>
+      <section class="card">
+        <img
+          src="assets/lightning-297608.svg"
+          alt="lighting bolt"
+          class="bookmark-selected"
+        />
+        <p>What position does Harry play on the Gryffindor Quidditch team?</p>
+        <p class="answer">Seeker</p>
+        <button class="button">SHOW ANSWER</button>
+
+        <ul class="tag-box">
+          <li class="tag"># Harry Potter</li>
+          <li class="tag"># Character</li>
+          <li class="tag"># Books</li>
+        </ul>
+      </section>
+
+      <section class="card">
+        <img
+          src="assets/lightning-297608.svg"
+          alt="lighting bolt"
+          class="bookmark-selected"
+        />
+        <p>Who is the Headmaster of Hogwarts when Harry arrives?</p>
+        <p class="answer visible">Albus Dumbledore</p>
+        <button class="button">HIDE ANSWER</button>
+
+        <ul class="tag-box">
+          <li class="tag"># Harry Potter</li>
+          <li class="tag"># Character</li>
+          <li class="tag"># Books</li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+      <nav class="navbar">
+        <ul>
+          <li>
+            <a href="./index.html">
+              <img
+                class="image"
+                src="assets/home_app_logo_FILL0_wght400_GRAD0_opsz24.svg"
+                alt="Home grafik"
+            /></a>
+          </li>
+          <li>
+            <a href="./bookmark.html">
+              <img
+                class="image image-select"
+                src="assets/offline_bolt_FILL0_wght400_GRAD0_opsz24.svg"
+                alt="Bookmark grafik"
```

Comment:
Is the last bookmark not selected here? I did this in such a way that only those cards appear in the bookmarks.html that also have a marked bookmark in the index.html. And the bookmarks.html can only contain cards with marked bookmarks, that's the logic behind it.

Reply:
Yes, you're right. I changed it. :-)